src/components/link/index.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+
+const COMPONENT_CLASS_NAME = 'c-link';
+
+const ExternalLink = ({ children, href }) => (
+	<a
+		className={COMPONENT_CLASS_NAME}
+		href={href}
+		rel="noreferrer"
+		target="_blank"
+	>
+		<span className="visually-hidden">Opens in new window</span>
+		{children}
+	</a>
+);
+
+const Link = ({ children, href, isExternal }) => {
+	// internal link by default
+	if (isExternal) {
+		return (
+			<ExternalLink href={href}>
+				{children}
+			</ExternalLink>
+		);
+	}
+
+	return (
+		<a
+			className={COMPONENT_CLASS_NAME}
+			href={href}
+		>
+			{children}
+		</a>
+	);
+};
+
+Link.propTypes = {
```

Comment:
We should ensure components have the ability for `className` to be passed in to allow for consumer to add their own

Reply:
gotcha, additional classnames